src/send/usePrepareSendTransactions.ts

```diff
@@ -18,32 +19,30 @@ export async function _prepareSendTransactionsCallback({
   token,
   recipientAddress,
   walletAddress,
-  isDekRegistered,
   feeCurrencies,
 }: {
   amount: BigNumber
   token: TokenBalance
   recipientAddress: string
   walletAddress: string
-  isDekRegistered: boolean
   feeCurrencies: TokenBalance[]
 }) {
-  const includeRegisterDekTx = !isDekRegistered && tokenSupportsComments(token)
-
   if (amount.isLessThanOrEqualTo(0)) {
     return
   }
   if (tokenBalanceHasAddress(token)) {
-    if (!includeRegisterDekTx) {
-      return prepareERC20TransferTransaction({
-        fromWalletAddress: walletAddress,
-        toWalletAddress: recipientAddress,
-        sendToken: token,
-        amount: BigInt(tokenAmountInSmallestUnit(amount, token.decimals)),
-        feeCurrencies,
-      })
+    const transactionParams = {
+      fromWalletAddress: walletAddress,
+      toWalletAddress: recipientAddress,
+      sendToken: token,
+      amount: BigInt(tokenAmountInSmallestUnit(amount, token.decimals)),
+      feeCurrencies,
+    }
+    if (tokenSupportsComments(token)) {
+      return prepareTransferWithCommentTransaction(transactionParams)
```

Comment:
this won't include comments because we only request it on the next screen right? Can fees change based on the length of the comment?

Reply:
yeah they can... I'd like to follow up on that in a separate PR since I think this one has a reasonable behavior for now

Reply:
it's a good point though

Reply:
one thing we could do is set a maximum length on comments on the next screen, and on this screen use a dummy comment that is of the max length, which is the most pessimistic scenario for gas fee estimation

Reply:
it's a little tricky because we then encrypt the comment, so we can't just use the same constant in both places